src/subgraph/actions/getMostRecentDomains.ts

```diff
@@ -0,0 +1,36 @@
+import { ApolloClient } from "@apollo/client/core";
+import { Domain } from "../../types";
+import * as queries from "../queries";
+import { DomainsQueryDto } from "../types";
+import { convertDomainDtoToDomain, performQuery } from "./helpers";
+
+export const getMostRecentDomains = async <T>(
+  apolloClient: ApolloClient<T>,
+  count: number = 100
+): Promise<Domain[]> => {
+  const domains: Domain[] = [];
+  let skip = 0;
```

Comment:
Always 0, can we just remove this from the query

Reply:
not true - on line 27 we add the results of the query to `skip`

Reply:
I can't read.

Reply:
I think there is a bug here with this though because this gets used as `indexId_gt: $startIndex`

indexId starts at 0 and increases for every domain created

but this query should return the highest indexId, and then return lower id's